test/express/middleware-custom-formatters-test.js

```diff
@@ -21,8 +21,9 @@ describe('Simple server - custom formatters', () => {
             .expect(400)
             .end((_err, res) => {
                 expect(res.body).to.eql({
-                    'more_info': '[{"keyword":"type","dataPath":".test","schemaPath":"#/properties/test/type","params":{"type":"object"},"message":"should be object"}]',
-                    'extra_text': 'dummy-text'
+                    'errors': '[{"keyword":"type","dataPath":".test","schemaPath":"#/properties/test/type","params":{"type":"object"},"message":"should be object"}]',
```

Comment:
maybe it's worth to put this out in some mockdata constant so it it can be easily changed and updated when needed?

Reply:
Why would this ever change if it's just a custom formatter created for the sake of the test?

Reply:
perhaps it looks like a pre-mature refactoring but take a look at `middleware-test.js`, a file with over 1000 lines of test code, which we can debate whether that's maintainable or not, but regardless, if you look at tests in that file - many of which refer to the `/pets` endpoint. If tomorrow morning, for some reason we need to change that to `/leprecon` it would be a search/replace instead of a constant string in one place.

As I'm not too familiar with the tests for the project, perhaps I picked a bad example with regards to the endpoint and that could be any random string but I assume the overall message is understood.

Reply:
My point is that not only premature, but unlikely ever tobe reused. This is a one-off assertion that is not going to be useful for other tests. For other tests - I agree that they are hard to maintain at this point, but I would start refactoring them by splitting.

Reply:
Sounds good 👍

Reply:
@lirantal I don't mind doing the change if you see value in it, though, as it's a trivial one. What do you think?

Reply:
Whichever you think is ok, I was merely pointing it out :-)